src/GT.Core/Services/Interfaces/IGTListingService.cs

```diff
@@ -1,19 +1,47 @@
 ﻿using GT.Core.DTO.Impl;
 using GT.Core.FilterModels.Interfaces;
+using GT.Data.Data.GTAppDb.Entities;
 
 namespace GT.Core.Services.Interfaces
 {
 	public interface IGTListingService : IGTService
 	{
 		/// <summary>
-		/// Gets partial listings for list view in UI.
+		/// Gets a list of listings meant for list view in UI, where some properties have been stripped from the view model.
 		/// </summary>
-		/// <param name="filter"></param>
+		/// <param name="filter">Optional filter paramater.</param>
+		/// <returns>Returns all listings which match the provided filtered values. Returns all listings if no filter is provided.</returns>
 		Task<List<ListingPartialDTO>> GetAsync(IListingFilterModel? filter = null);
```

Comment:
Should ListingPartialDTO be renamed per @KristofferHogberg comment that it is ambiguous naming? Proposal ListingOverviewDTO or similar.

Reply:
Fix in commit.